src/sleeplocks.erl

```diff
@@ -154,7 +154,12 @@ handle_cast(_Msg, Lock) ->
     {noreply, Lock}.
 
 %% @hidden
-%% Empty shim to implement behaviour.
+%% Handles releasing locks if owners crash.
+handle_info({'DOWN', _Ref, process, Pid, _Reason}, #lock{current = Current} = Lock) ->
+    {noreply, Lock#lock{current = maps:remove(Pid, Current)}};
+
+%% @hidden
+%% Fallback for non-matching info.
```

Comment:
This looks fine, but please can you make this an addition only? There is no need to change the existing comment.

Reply:
My reasoning here is that I'm extending `handle_info/2` with a new clause. Therefore, the code is now using `handle_info/2`, which makes the comment "Empty shim to implement behaviour." not really current anymore.

But: I don't want the locks gen_server to crash if someone were to send it a random message, which is why I left that extra all-matching clause in.